trulens_eval/trulens_eval/feedback/__init__.py

```diff
@@ -6,7 +6,7 @@
 # Providers of feedback functions evaluation:
 from trulens_eval.feedback.provider.hugs import Huggingface
 from trulens_eval.feedback.provider.langchain import Langchain
-from trulens_eval.utils.imports import OptionalImports
+from trulens_eval.utils.imports import REQUIREMENT_CORTEX, OptionalImports
```

Comment:
the imports of this file seem to not use commas, so you probably should keep that going.